Documentation/Troubleshooting/ceph-csi-common-issues.md

```diff
@@ -413,9 +413,13 @@ Where `-m` is one of the mon endpoints and the `--key` is the key used by the CS
 
 When a node is lost, you will see application pods on the node stuck in the `Terminating` state while another pod is rescheduled and is in the `ContainerCreating` state.
 
-To allow the application pod to start on another node, force delete the pod.
+### For cluster with kubernetes version greater than 1.26
```

Comment:
Instead of changing the headers, let's just add a note such as:
```suggestion
!!! important
    For clusters with Kubernetes version 1.26 or greater, see the [improved automation](../Storage-Configuration/Block-Storage-RBD/block-storage.md#recover-rbd-rwo-volume-in-case-of-node-loss) to recover from the node loss. If using K8s 1.25 or older, continue with these instructions.
```

Then the rest of the changes in this doc can be reverted.